launch/index.ts

```diff
@@ -0,0 +1,78 @@
+import { options } from "@parallel-finance/api";
```

Comment:
let's use single quote maybe, we can also edit `.editorconfig` to add some rules